Hue normal shading in heightfield2shading (#412)
DGtal-team/DGtalTools: ChangeLog.md

# DGtalTools 1.3 (beta)

- *converters*

>   - heightfield2shading: new option to add an hue based rendering (from normal

Reviewer: ```suggestion
  - heightfield2shading: new option to add a matcap rendering (from normal
```
